src/main/scala/intent/sbt/Runner.scala

```diff
@@ -9,9 +9,9 @@ import java.io.{PrintWriter, StringWriter}
 
 class Framework extends SFramework with
   def name(): String = "intent"
-  def fingerprints(): Array[Fingerprint] = Array(IntentFingerprint)
-  def runner(args: Array[String], remoteArgs: Array[String], testClassLoader: ClassLoader): Runner =
-    new SbtRunner(args, remoteArgs, testClassLoader)
+  def fingerprints(): Array[Fingerprint | Null] = Array(IntentFingerprint)
+  def runner(args: Array[String | Null] | Null, remoteArgs: Array[String | Null] | Null, testClassLoader: ClassLoader | Null): Runner =
+    new SbtRunner(args.nn, remoteArgs.nn, testClassLoader.nn)
```

Comment:
**Overridden java arguments are never null**
Argument should never be null, immediately remove nullness of parameter.